sentry-android-ndk/build.gradle.kts

```diff
@@ -36,6 +36,7 @@ android {
 
         externalNativeBuild {
             cmake {
+                targets.add("sentry-android")
```

Comment:
that's not right, I'll need to duplicate this line elsewhere, this should be fixed on source origin.